src/test/java/com/orzechazo/eshop/services/UserServiceImplIT.java

```diff
@@ -0,0 +1,142 @@
+package com.orzechazo.eshop.services;
+
+import com.orzechazo.eshop.bootstrap.tests.BootstrapUsersAndOrders;
+import com.orzechazo.eshop.domain.Order;
+import com.orzechazo.eshop.domain.User;
+import com.orzechazo.eshop.domain.dto.UserDto;
+import com.orzechazo.eshop.exceptions.BadRequestException;
+import com.orzechazo.eshop.exceptions.ResourceNotFoundException;
+import com.orzechazo.eshop.repositories.OrderRepository;
+import com.orzechazo.eshop.repositories.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SpringExtension.class)
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class UserServiceImplIT {
+    @Autowired
+    private UserRepository userRepository;
+    @Autowired
+    private OrderRepository orderRepository;
+    private UserServiceImpl userService;
+    private final static String DB_USER_LOGIN = BootstrapUsersAndOrders.DB_USER_LOGIN;
+    private int DEFAULT_DB_USER_COUNT;
+    private BootstrapUsersAndOrders bootstrap;
+    @BeforeEach
+    void setUp() {
+        bootstrap = new BootstrapUsersAndOrders(orderRepository, userRepository);
+        bootstrap.loadData();
+        DEFAULT_DB_USER_COUNT = bootstrap.getUsers().size();
+
+        userService = new UserServiceImpl(userRepository);
+    }
+
+    @Test
+    void getAllUsers() {
+        List<String> dbUsersLoginList = bootstrap.getUsers().stream()
+                .map(User::getLogin).toList();
+        //when
+        List<UserDto> userDtos = userService.getAllUsers();
+        List<String> userDtosLogins = userDtos.stream().map(UserDto::getLogin).toList();
+        //then
+        assertThat(dbUsersLoginList).containsExactlyInAnyOrderElementsOf(userDtosLogins);
+    }
+
+    @Test
+    void getUserByLogin() {
+        UserDto returnedDto = userService.getUserDtoByLogin(DB_USER_LOGIN);
+
+        assertEquals("user1",returnedDto.getLogin());
+        assertNull(returnedDto.getPassword());
```

Comment:
You are checking here two different functionalities - in this test just make assertion for user login, and regarding password create separate test, I was thinking for a second why in 'getUserByLogin' test we are making sure that password is null.

Create test like shouldNotMapPassword and checking it there.